poll/translations/config.yaml

```diff
@@ -13,6 +13,7 @@ locales:
     - pt_BR  # Portuguese (Brazil)
     - ru  # Russian
     - zh_CN  # Chinese (China)
+    - de  # German
```

Comment:
Shouldn't this be `de_DE` as [edx-platform](https://github.com/edx/edx-platform/blob/master/conf/locale/config.yaml#L22)?

Reply:
Ok I will update it.

Reply:
@umarmughal824 Insofar as these comments apply to the other translation PRs, please update those as well.

Reply:
@xitij2000 Yes, I have commented on these other PR's please take a look.